app/shared_content/sensor_types_view.sql

```diff
@@ -0,0 +1,3 @@
+-- View for sensor types full data
+create or replace view shared_content.sensor_types_view
+    as select * from package_shared.sensor_types_view;
```

Comment:
Each of the files in `app/shared_content` are one liners. Do we need separate files for them?
Also, all three look like they are missing grant statements: `grant read on ...  to app role ...`. Do we not need those?

Reply:
I think one liners are ok, as they keep the main setup_script at the same granularity level (schema + file includes).

As for the permissions, these are not there in the original setup script. @sfc-gh-cgorrie , do you have more info on that?